extensions/common/vault/vault-aws/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2020, 2021, 2022 Amazon Web Services
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Amazon Web Services - initial API and implementation
+ *
+ */
+
+plugins {
+    `java-library`
+}
+
+dependencies {
+    api(project(":spi:common:core-spi"))
+    implementation("software.amazon.awssdk:secretsmanager:2.19.3")
```

Comment:
please provide a PR on the `GradlePlugins` repository to add this library version to the version-catalog so we can use it here.
https://github.com/eclipse-edc/GradlePlugins/blob/main/gradle/libs.versions.toml